src/shared/compatibility.js

```diff
@@ -14,13 +14,18 @@
  */
 /* eslint no-var: error */
 
-const { globalScope, } = require('./global_scope');
-
 // Skip compatibility checks for modern builds and if we already ran the module.
 if ((typeof PDFJSDev === 'undefined' || !PDFJSDev.test('SKIP_BABEL')) &&
-    !globalScope._pdfjsCompatibilityChecked) {
+    (typeof globalThis === 'undefined' ||
+     !globalThis._pdfjsCompatibilityChecked)) {
 
-globalScope._pdfjsCompatibilityChecked = true;
+// Provides support for globalThis in legacy browsers.
+// Support: IE.
```

Comment:
This should also include Edge and Opera since both don't support this feature and both browsers are supported according to our FAQ.

Reply:
Sure, fixed now :-)